src/main/java/com/sap/sgs/phosphor/fosstars/model/feature/OwaspDependencyCheckUsageFeature.java

```diff
@@ -0,0 +1,38 @@
+package com.sap.sgs.phosphor.fosstars.model.feature;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.sap.sgs.phosphor.fosstars.model.Value;
+import com.sap.sgs.phosphor.fosstars.model.value.OwaspDependencyCheckUsage;
+import com.sap.sgs.phosphor.fosstars.model.value.OwaspDependencyCheckUsageValue;
+import java.util.Objects;
+
+/**
+ * A feature which holds a {@link OwaspDependencyCheckUsage} value. Usage status mandatory,
+ * optional or not found.
```

Comment:
Maybe it's going to have more or less elements. Therefore, the javadoc would need to be updated. Let's just drop it.

```suggestion
 * A feature which holds a {@link OwaspDependencyCheckUsage} value.
```